features/step_definitions/access_control_steps.rb

```diff
@@ -9,13 +9,13 @@
 end
 
 Given(/^there are manuals created by multiple organisations$/) do
-  login_as(:cma_editor)
+  login_as(:gds_editor)
```

Comment:
I think it might be better to introduce a `generic_editor_from_another_organisation` factory or similar to avoid overloading the use of `gds_editor` and I think this would make the intended behaviour clearer.